src/libs/actions/Report.ts

```diff
@@ -1027,12 +1035,14 @@ function navigateToAndOpenReport(
 
     // We want to pass newChat here because if anything is passed in that param (even an existing chat), we will try to create a chat on the server
     openReport(report?.reportID ?? '', '', userLogins, newChat, undefined, undefined, undefined, avatarFile);
-    if (shouldDismissModal) {
-        Navigation.dismissModalWithReport(report);
-    } else {
-        Navigation.navigateWithSwitchPolicyID({route: ROUTES.HOME});
-        Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(report?.reportID ?? '-1'), actionType);
-    }
+    InteractionManager.runAfterInteractions(() => {
+        if (shouldDismissModal) {
+            Navigation.dismissModalWithReport(report);
+        } else {
+            Navigation.navigateWithSwitchPolicyID({route: ROUTES.HOME});
+            Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(report?.reportID ?? '-1'), actionType);
+        }
+    });
```

Comment:
Why are we adding the `InteractionManager`. here? We should wait for interaction only after we register them. Doing so without any registered handler is a workaround

Reply:
Is this bug related to the new feature (group avatar view) or an existing bug?

Reply:
I'm still not sure what went wrong here.

> The problem occurs because openReport from the report screen is triggered before the new group report data is stored in Onyx

Why this does not happen when we proceed with the flow normally? (without the avatar view feature)